META.json

```diff
@@ -0,0 +1,93 @@
+{
+   "abstract" : "Encapsulate system info for OpenMP",
+   "author" : [
+      "oodler <oodler@cpan.org>"
+   ],
+   "dynamic_config" : 1,
+   "generated_by" : "Dist::Zilla version 6.022, CPAN::Meta::Converter version 2.150010",
+   "license" : [
+      "perl_5"
+   ],
+   "meta-spec" : {
+      "url" : "http://search.cpan.org/perldoc?CPAN::Meta::Spec",
+      "version" : 2
+   },
+   "name" : "Alien-OpenMP",
+   "no_index" : {
+      "directory" : [
+         "eg",
+         "examples",
+         "inc",
+         "share",
+         "t",
+         "xt"
+      ]
+   },
+   "prereqs" : {
+      "build" : {
+         "requires" : {
+            "Alien::Build" : "0.32",
+            "Alien::Build::MM" : "0.32",
+            "ExtUtils::MakeMaker" : "6.52"
+         }
+      },
+      "configure" : {
+         "requires" : {
+            "Alien::Build" : "1.40",
+            "Alien::Build::MM" : "0.32",
+            "ExtUtils::CBuilder" : "0",
+            "ExtUtils::MakeMaker" : "6.52"
+         }
+      },
+      "develop" : {
+         "requires" : {
+            "File::Spec" : "0",
+            "IO::Handle" : "0",
+            "IPC::Open3" : "0",
+            "Test::More" : "0",
+            "Test::Pod" : "1.41"
+         }
+      },
+      "test" : {
+         "recommends" : {
+            "CPAN::Meta" : "2.120900"
+         },
+         "requires" : {
+            "ExtUtils::MakeMaker" : "6.52",
+            "File::Spec" : "0",
+            "Test::More" : "0"
+         }
+      }
+   },
+   "provides" : {
+      "Alien::OpenMP" : {
+         "file" : "lib/Alien/OpenMP.pm",
+         "version" : "0.003004"
+      },
+      "Alien::OpenMP::configure" : {
+         "file" : "lib/Alien/OpenMP/configure.pm",
+         "version" : "0.003004"
+      }
+   },
+   "release_status" : "stable",
+   "resources" : {
+      "homepage" : "https://github.com/Grinnz/p5-Alien-OpenMP",
+      "repository" : {
+         "type" : "git",
+         "url" : "https://github.com/Grinnz/p5-Alien-OpenMP.git",
```

Comment:
This doesn't look right?

Reply:
Good catch, but they should get corrected once someone with remote set to the upstream repo does dzil regenerate or dzil release.